src/app/styles/variables.scss

```diff
@@ -0,0 +1,11 @@
+$spacing-xs: 0.25rem;
+$spacing-s: 0.5rem;
+$spacing-m: 1rem;
+$spacing-l: 1.5rem;
+$spacing-xl: 2.5rem;
+$border-radius: 0.5rem;
+$max-width-main: 1440px;
+$breakpoint-tablet: 900px;
+$breakpoint-phone: 599px;
+
```

Comment:
To make it a bit more readable please separate these variables with a whitespace.

```
$spacing-xs: 0.25rem;
$spacing-s: 0.5rem;
$spacing-m: 1rem;
$spacing-l: 1.5rem;
$spacing-xl: 2.5rem;

$border-radius: 0.5rem;

$max-width-main: 1440px;
$breakpoint-tablet: 900px;

$breakpoint-phone: 599px;

$transition: ease-in-out 0.3s;
```